examples/pygstc/simple_pipeline.py

```diff
@@ -0,0 +1,180 @@
+          videoconvert ! videoscale ! autovideosink"
+
+    def playVideo(self):
+        print("Video playing")
+        if (not self.pipe_exists(self.pipeName)):
+            # Create pipeline object
+            self.gstc.pipeline_create(self.pipeName, self.pipeline)
+            self.gstc.pipeline_play(self.pipeName)
+            self.thErrorHandler = threading.Thread(
+                                    target=self.errPlayerHandler,
+                                    args=())
+            self.thErrorHandler.start()
+
+        else:
+            # Continue playing after pause
+            self.gstc.pipeline_play(self.pipeName)
+
+    def pauseVideo(self):
+        print("Video paused")
+        self.gstc.pipeline_pause(self.pipeName)
+
+    def stopVideo(self):
+        print("Video stopped")
+        if (self.pipe_exists(self.pipeName)):
+            self.gstc.pipeline_stop(self.pipeName)
+            self.gstc.pipeline_delete(self.pipeName)
+            self.thErrorHandler.join()
+
+    def setSpeed(self, speed):
+        print("Setting play speed to: "+str(speed))
+        self.playingSpeed = speed
+        self.gstc.event_seek(self.pipeName,
+                             rate=self.playingSpeed,
+                             format=3,
+                             flags=1,
+                             start_type=1,
+                             start=0,
+                             end_type=1,
+                             end=-1)
+
+    def jumpTo(self, position):
+        print("Jump to time: "+str(position))
+        self.gstc.event_seek(self.pipeName,
+                             rate=float(self.playingSpeed),
+                             format=3,
+                             flags=1,
+                             start_type=1,
+                             start=(position*(1000000000)),
+                             end_type=0,
+                             end=-1)
+
+    def pipe_exists(self, pipe_name):
+        # Check if pipe is already created
+        existing_pipes = self.gstc.list_pipelines()
+        if (existing_pipes == []):
+            ret = False
+        elif(existing_pipes[0]['name'] == pipe_name):
+            ret = True
+        else:
+            ret = False
+        return ret
+
+
+def printUsage():
+    print("play: To play and run")
+    print("pause: To pause the video")
+    print("stop: To stop and close the playing")
+    print("set_speed $SPEED")
+    print("jump $TIME [in seconds]")
+
+
+def printError():
+    print("To use run: python3 simple_pipeline.py $VIDEO_PATH",
+          file=sys.stderr)
+
+
+if __name__ == "__main__":
+    print("Sample PyGstC Video Player")
+
+    if(len(sys.argv) != 2):
+        printError()
+        sys.exit(0)
+    try:
+        myPlayer = GstcPlayer()
+
+        if(not myPlayer.openVideo(sys.argv[1])):
+            print("Pipeline already exits", file=sys.stderr)
+            sys.exit(0)
+
+        action = ["running"]
```

Comment:
This is confusing? Why running? If it is to have a non-used word I recommend using `None` instead.